src/components/in-world-hud.js

```diff
@@ -7,6 +7,12 @@ AFRAME.registerComponent("in-world-hud", {
     this.mic = this.el.querySelector(".mic");
     this.freeze = this.el.querySelector(".freeze");
     this.bubble = this.el.querySelector(".bubble");
+    this.background = this.el.querySelector(".bg");
+    const renderOrder = window.APP.RENDER_ORDER;
+    this.mic.object3DMap.mesh.renderOrder = renderOrder.HUD;
```

Comment:
Nit, but at first glance I was confused why these all were teh same, I would maybe call it HUD_CONTNET or HUD_ICONS